docs/event.md

```diff
@@ -35,7 +35,7 @@ Creates a set of events for the given domElement that can be handled through a c
 
 ### Arguments:
 
-1. app  - (*element*) A [LumaGL application](core.html#LumaGL:constructor).
+1. gl  - (*object*) A WebGL context. Events are handled for the context's canvas element.
```

Comment:
The WebGL objects all have well defined types. In this case [WebGLRenderingContext](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext)

Let's embrace that and use it in our documtenation.